src/humanloop/otel/processor/prompts.py

```diff
@@ -1,4 +1,4 @@
-import deepdiff
+import deepdiff  # type: ignore [import]
```

Comment:
is this necessary? or did we lose types from the version downgrade?

Reply:
good point - the downgrade is not required. reverted deepdiff to higher version, removing the need for the ignore